pandas/tests/groupby/test_counting.py

```diff
@@ -212,3 +212,13 @@ def test_count_with_datetimelike(self, datetimelike):
         expected = DataFrame({'y': [2, 1]}, index=['a', 'b'])
         expected.index.name = "x"
         assert_frame_equal(expected, res)
+
+    def test_count_with_only_nans_in_first_group(self):
+        # GH21956
+        df = DataFrame({'A': [np.nan, np.nan], 'B': ['a', 'b'], 'C': [1, 2]})
+        result = df.groupby(['A', 'B']).C.count()
```

Comment:
can u do an assert_produces_warning()

here (should not show a warning)

Reply:
This does not produce any warnings for me.

What should look for in travis? Grep "SeriesGroupBy.count"?

Reply:
Ok, will look in the morning.

Reply:
ok, can you assert that this produces no warning (e.g. it IS producing a warning now in numpy < 1.13)